SU-HSDO/suhumsci
# Hsd8 268 run importer ui

# READY FOR REVIEW

# Summary
- Provide a simple path like D7 to allow users to run importers
- Granular permissions.

# Needed By (Date)
- 12/5

# Urgency
- medium

# Steps to Test

1. checkout this branch
1. sync a site `blt drupal:sync --site=[sitename] --partial`
1. go to /import page
1. review the list of migration importers
1. run one of the importers
1. verify no errors occur.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
